src/main/java/org/apache/sling/feature/cpconverter/handlers/DefaultEntryHandlersManager.java

```diff
@@ -30,10 +31,10 @@
     private final List<EntryHandler> entryHandlers = new LinkedList<>();
 
     public DefaultEntryHandlersManager() {
-        this(Collections.emptyMap(), false);
+        this(Collections.emptyMap(), false, SlingInitialContentPolicy.KEEP);
     }
 
-    public DefaultEntryHandlersManager(@NotNull Map<String, String> configs, boolean enforceConfigurationsAndBundlesBelowProperFolder) {
+    public DefaultEntryHandlersManager(@NotNull Map<String, String> configs, boolean enforceConfigurationsAndBundlesBelowProperFolder, SlingInitialContentPolicy slingInitialContentPolicy) {
         ServiceLoader<EntryHandler> entryHandlersLoader = ServiceLoader.load(EntryHandler.class);
         for (EntryHandler entryHandler : entryHandlersLoader) {
             if (configs.containsKey(entryHandler.getClass().getName())) {
```

Comment:
@kwin, we have an issue here - the special casing will below will only work if there is a config for a handler. In case of the BundleEntryHandler (e.g.) there is none by default - hence, we never set the policy (and possibly don't for the other cases either).

Reply:
Should be fixed by https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/2f9af87685fc84ce63454fa094336350d642cae5

Reply:
Works now. I like the change for the`--sling-initial-content-policy` - thanks a lot! Can you update the README with it so we don't forget?

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/88bb353f6e2bc6ceec24c8e60d49e5bb0c1e93f0.